packages/components/src/PlaceCard/Image/ImageTypes.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import type { StylePropType } from '@kiwicom/universal-components';
+
+export type Props = {|
+  +source: {|
+    uri: string,
+    height?: number,
+    width?: number,
```

Comment:
these props should also be read-only

Reply:
I am afraid this is not possible because of `react-native-web` flow-typed types:
https://github.com/kiwicom/margarita/blob/383ebc262ac22ce0132f1466344caacde604a0ad/flow-typed/npm/react-native-web_vx.x.x.js#L317
>    source?: string | {| uri: string, width?: number, height?: number |},

Reply:
🙈

Reply:
But this is checked in to our git, so we could always change it to fit our way of using flow.

Reply:
But when we upgrade version, it might be painful. Well, I guess it could be painful any way.

Reply:
Done.